.github/workflows/build-on-release.yml

```
@@ -20,20 +17,135 @@ jobs:
+        name: krr-ubuntu-latest-${{ github.ref_name }}
+        path: ./krr-ubuntu-latest-${{ github.ref_name }}.zip
+
+    - name: Calculate hash
+      id: calc-hash
+      run: echo "LINUX_BUILD_HASH=$(sha256sum krr-ubuntu-latest-${{ github.ref_name }}.zip | awk '{print $1}')" >> $GITHUB_OUTPUT
+
+  build-windows:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)